Directory.Build.props

```
@@ -21,6 +21,10 @@
     <BaseIntDir Condition="'$(Platform)' != 'Win32'">$(RootIntDir)\$(Platform)\$(Configuration)</BaseIntDir>
     <BaseOutDir Condition="'$(Platform)' == 'Win32'">$(RootOutDir)\x86\$(Configuration)</BaseOutDir>
     <BaseOutDir Condition="'$(Platform)' != 'Win32'">$(RootOutDir)\$(Platform)\$(Configuration)</BaseOutDir>
+
+    <ReactNativeRootIntDir>$(ReactNativeWindowsDir)build\</ReactNativeRootIntDir>
+    <ReactNativeRootOutDir>$(ReactNativeWindowsDir)target\</ReactNativeRootOutDir>   
+    <ReactNativeYarnIntDir>$(ReactNativeRootIntDir)yarn_install\</ReactNativeYarnIntDir>
```

Just use the existing props.

```suggestion
    <ReactNativeYarnIntDir>$(RootIntDir)yarn_install\</ReactNativeYarnIntDir>
```

---

Good point! I can address the issue with prefixes in future PRs. Remove the duplicates for now.